src/USER-PHONON/dynamical_matrix.cpp

```diff
@@ -0,0 +1,482 @@
+//
+// Created by charlie sievers on 6/21/18.
+//
+
+#include <mpi.h>
+#include <stdlib.h>
+#include <iostream>
+#include "dynamical_matrix.h"
+#include "atom.h"
+#include "modify.h"
+#include "domain.h"
+#include "comm.h"
+#include "group.h"
+#include "force.h"
+#include "math_extra.h"
+#include "memory.h"
+#include "bond.h"
+#include "angle.h"
+#include "dihedral.h"
+#include "improper.h"
+#include "kspace.h"
+#include "update.h"
+#include "neighbor.h"
+#include "pair.h"
+#include "timer.h"
+#include "finish.h"
+#include "ctime"
```

Comment:
Please do not use this include (and it should be `<ctime>` not `"ctime"` anyway). clock() wraps around frequently on 32-bit platforms. We use MPI_Wtime() for measuring time for special cases, and use the `Timer` class for general measurements, where we want to get the decomposition into different sections (potentials, communication, neighbor lists etc.), e.g. in time integration and minimization.